application/views/notes/add.php

```diff
@@ -1,52 +1,60 @@
-
+<!-- Notes add view: form for creating a new note -->
 <div class="container notes">
-
-<div class="card">
-  <div class="card-header">
-    <h2 class="card-title"><?= __("Create Note"); ?></h2>
-    <ul class="nav nav-tabs card-header-tabs">
-	    <li class="nav-item">
-	    	<a class="nav-link" href="<?php echo site_url('notes'); ?>"><?= __("Notes"); ?></a>
-	    </li>
-	    <li class="nav-item">
-	    	<a class="nav-link active" href="<?php echo site_url('notes/add'); ?>"><?= __("Create Note"); ?></a>
-	    </li>
-	</ul>
-  </div>
-
-  <div class="card-body">
-
-  	<?php if (!empty(validation_errors())): ?>
-    <div class="alert alert-danger">
+  <div class="card">
+    <div class="card-header">
+      <h2 class="card-title"><?= __("Notes"); ?></h2>
+      <ul class="nav nav-tabs card-header-tabs">
+        <li class="nav-item">
+          <a class="nav-link" href="<?= site_url('notes'); ?>">
+            <i class="fa fa-sticky-note"></i> <?= __("Your Notes"); ?>
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link active" href="<?= site_url('notes/add'); ?>">
+            <i class="fa fa-plus-square"></i> <?= __("Create Note"); ?>
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div class="card-body">
+      <!-- Show validation errors if any -->
+      <?php if (!empty(validation_errors())): ?>
+      <div class="alert alert-danger">
         <a class="btn-close" data-bs-dismiss="alert" title="close">x</a>
         <ul><?php echo (validation_errors('<li>', '</li>')); ?></ul>
+      </div>
+      <?php endif; ?>
+      <!-- Note creation form -->
+      <form method="post" action="<?php echo site_url('notes/add'); ?>" name="notes_add" id="notes_add">
+        <div class="mb-3">
+          <label for="inputTitle"><?= __("Title"); ?></label>
+          <input type="text" name="title" class="form-control" id="inputTitle" value="<?php echo isset($suggested_title) && $suggested_title ? $suggested_title : set_value('title'); ?>">
+        </div>
+        <div class="mb-3">
+          <label for="catSelect">
+            <?= __("Category"); ?>
+            <span class="ms-1" data-bs-toggle="tooltip" title="<?= __("Contacts") . __(" is a special note category used in various places of Wavelog to store information about QSO partners. This notes are private and are not shared with other users nor exported to external services.") ?>">
```

Comment:
Same as in application/views/notes/edit.php Line 43. Don't rip sentences apart


```suggestion
            <span class="ms-1" data-bs-toggle="tooltip" title="<?= __("Contacts is a special note category used in various places of Wavelog to store information about QSO partners. This notes are private and are not shared with other users nor exported to external services.") ?>">
```